aten/src/ATen/native/mkldnn/TensorFactories.cpp

```diff
@@ -4,7 +4,7 @@ namespace at { namespace native {
 
 #if AT_MKLDNN_ENABLED()
 
-Tensor empty_mkldnn(IntArrayRef sizes, const TensorOptions& options) {
+Tensor empty_mkldnn(IntArrayRef sizes, const TensorOptions& options, c10::optional<c10::MemoryFormat> optional_memory_format) {
```

Comment:
Shouldn't there be a check here that `optional_memory_layout` is `nullopt`?

Reply:
done